test/integration_test/test-deploy.sh

```diff
@@ -112,6 +112,12 @@ for i in $(seq 1 100) ; do
     test_status=$(kubectl -n kube-system get pod operator-test -o json | jq ".status.phase" -r)
     if [ "$test_status" == "Running" ] || [ "$test_status" == "Completed" ]; then
         break
```

Comment:
[nit], can we add this here?
```
echo "Tests succeeded"
```

Reply:
This isn't for checking if the tests succeeded: this is just to make sure that the pod has started (e.g. the tests are running). The next step in this script is to follow the logs of the pod.

At the [very end](https://github.com/libopenstorage/operator/pull/398/files#diff-812c21dc01db7e65cbf367911b1630f59a7191cd4a36649258c4e373c4a84e21R141) there's already a line printing out "Tests passed".